### PR TITLE
Adding more steps to operator installation test case

### PR DESCRIPTION
**Pull request by VeereshAradhya**
- Adding a step to verify the status of the tektoninstallersets
- Adding a step to verify if the default tektoninstallersets are present

### CODE REVIEW

File: specs/olm.spec [hunk 1]
```diff
@@ -20,6 +20,8 @@ Steps:
   * Verify TektonAddons Install status
   * Validate RBAC
   * Validate default auto prune cronjob in target namespace
+  * Validate tektoninstallersets
```
Comment (ppitonak) [inline specs/olm.spec:23]: I would rename it to `Validate tektoninstallersets state` so that it's clear why we have two separate steps

**Review by ppitonak — CHANGES_REQUESTED** [review]

File: pkg/k8s/k8s.go [hunk 2]
```diff
@@ -394,3 +394,50 @@ func AssertCronjobNotPresent(c *clients.Clients, cronJobName, namespace string)
...
+	tis, err := c.Operator.TektonInstallerSets().List(c.Ctx, metav1.ListOptions{})
+	failedInstallersets := make([]string, 0)
+	if err != nil {
+		assert.FailOnError(fmt.Errorf("Error getting tektoninstallersets: %v", err))
+	}
+
+	for _, is := range tis.Items {
+		log.Printf("Verifying if the installerset %s is in ready state", is.Name)
+		if !is.Status.IsReady() {
+			failedInstallersets = append(failedInstallersets, is.Name)
```
[the remark this answers is not in the document]
Comment (VeereshAradhya) [reply pkg/k8s/k8s.go:408]: Reason meaning, why the installerset is in failed state?

[the remark this answers is not in the document]
Comment (ppitonak) [reply pkg/k8s/k8s.go:408]: yes, whatever appears in reason column
```
$ oc get tektoninstallerset
NAME                                     READY   REASON
addon-clustertasks-t6tlz                 True
```

[the remark this answers is not in the document]
Comment (VeereshAradhya) [reply pkg/k8s/k8s.go:408]: I want to use API here instead of cli. Is it ok if I add this in the next PR?

[the remark this answers is not in the document]
Comment (VeereshAradhya) [reply pkg/k8s/k8s.go:408]: I am trying to get installerset to `Error` state to test this, but I am unable to do it

[the remark this answers is not in the document]
Comment (ppitonak) [reply pkg/k8s/k8s.go:408]: Sure, let's do it in new PR.

**Review by ppitonak — APPROVED** [review]

**Pull request merged**